Assets/LeapMotion/Scripts/Utils/Hands.cs

```diff
@@ -0,0 +1,169 @@
+      InitStatic();
+      SceneManager.activeSceneChanged += InitStaticOnNewScene;
+    }
+
+    private static void InitStaticOnNewScene(Scene unused, Scene unused2) {
+      InitStatic();
+    }
+
+    private static void InitStatic() {
+      s_provider = GameObject.FindObjectOfType<LeapServiceProvider>();
+      s_leapRig = s_provider.transform.parent.parent.parent.gameObject;
+    }
+
+    /// <summary>
+    /// Static convenience accessor for the Leap Rig GameObject.
+    /// </summary>
+    public static GameObject Rig {
+      get { return s_leapRig; }
+    }
+
+    public static Hand Get(Chirality chirality) {
+      if (chirality == Chirality.Left) return Left;
+      else return Right;
+    }
+
+    /// <summary>
+    /// Returns the first left hand found by Leap in the current frame, otherwise returns null if no such hand is found.
+    /// </summary>
+    public static Hand Left {
+      get {
+        List<Hand> hands = s_provider.CurrentFrame.Hands;
+        for (int i = 0; i < hands.Count; i++) {
+          if (hands[i].IsLeft) {
+            return hands[i];
+          }
+        }
+        return null;
+      }
+    }
+
+    /// <summary>
+    /// Returns the first right hand found by Leap in the current frame, otherwise returns null if no such hand is found.
+    /// </summary>
+    public static Hand Right {
+      get {
+        List<Hand> hands = s_provider.CurrentFrame.Hands;
+        for (int i = 0; i < hands.Count; i++) {
+          if (hands[i].IsRight) {
+            return hands[i];
+          }
+        }
+        return null;
+      }
+    }
+
+    /// <summary>
+    /// Returns the thumb object for the given Hand.
+    /// </summary>
+    public static Finger Thumb(this Hand hand) {
+      return hand.Fingers[(int)Leap.Finger.FingerType.TYPE_THUMB];
+    }
+
+    /// <summary>
+    /// Returns the index Finger object for the given Hand.
+    /// </summary>
+    public static Finger Index(this Hand hand) {
+      return hand.Fingers[(int)Leap.Finger.FingerType.TYPE_INDEX];
+    }
+
+    /// <summary>
+    /// Returns the middle Finger object for the given Hand.
+    /// </summary>
+    public static Finger Middle(this Hand hand) {
+      return hand.Fingers[(int)Leap.Finger.FingerType.TYPE_MIDDLE];
+    }
+
+    /// <summary>
+    /// Returns the ring Finger object for the given Hand.
+    /// </summary>
+    public static Finger Ring(this Hand hand) {
+      return hand.Fingers[(int)Leap.Finger.FingerType.TYPE_RING];
+    }
+
+    /// <summary>
+    /// Returns the pinky Finger object for the given Hand.
+    /// </summary>
+    public static Finger Pinky(this Hand hand) {
+      return hand.Fingers[(int)Leap.Finger.FingerType.TYPE_PINKY];
+    }
+
+    /// <summary>
+    /// Returns the direction the Hand's palm is facing.
+    /// For the other two axis directions, see ThumbAxis and FingerAxis.
+    /// </summary>
+    public static Vector3 PalmAxis(this Hand hand) {
+      return -hand.Basis.yBasis.ToVector3();
```

Comment:
I agree that FingerAxis, ThumbAxis, PalmNormal would not be good either, but maybe we could come up with some different names for ThumbAxis and FingerAxis that would make sense?